### ISSUE TITLE
Add ability to run only one scenario within single test

### ISSUE THREAD

**Issue by dryobates** [kidosoft/Morelia#16]
As a common request there should be possibility to run only one scenario within TestCase. Usage similar to that below should be sufficient:

```python
class CalculatorTestCase(unittest.TestCase):

    def test_addition(self):
        """ Addition feature """
        run('calculator.feature', self, scenario="Add two numbers", verbose=True)
```

**Comment by Mathumathisriram** [comment]
hi,
May I know when this can be done.

**Comment by izabeera** [comment]
Hi,
We didn't consider it as the highest priority, but if you need it we can reprioritize  our tasks. Or... Maybe you would like to do it yourself? We can guide you :)

**Comment by Mathumathisriram** [comment]
it would be great If you can make this done. Thanks

**Comment by Deelipan** [comment]
I'm also expecting this feature to be done. Hope that it will happen soon. Thanks

**Comment by Mathumathisriram** [comment]
Hi Can you please tell me, how to run only one scenario within the test case.......?

**Comment by midnighteuler** [comment]
See https://github.com/kidosoft/Morelia/pull/89

**Comment by dryobates** [comment]
Released in 0.7.1
